### ISSUE TITLE
Problems adding a new group test and commands.

### ISSUE THREAD

**Issue by vteixeira19** [dsheiko/puppetry#9]
Hello,

I'm Q&A manager and I'm trying use Puppetry in my tests at work.  

But after creating a new project, I can not add test or anything else.

Check what happen in this video: https://cl.ly/bc4d1d81a19d

Thank you for your time.

**Comment by dsheiko** [comment]
First of all. thank you very much for reporting. I did it a quick check, could not reproduce the issue straight away. Can you please send me directory with your project where you have the crash? BTW, what platform do you use? Windows. Linux or MacOS?

**Comment by vteixeira19** [comment]
Hello,

I was checking and realized that the error arises when I try to write in
the input 'Method': https://cl.ly/fdaa062aebb1 . I was writing 'Visite' in
the input 'Method' instead of select 'Goto'.
If I don't write the same error no longer happens. The plataform I use is
MacOS.

Thank you.

Best regards.
Vanessa Teixeira


Dmitry Sheiko <notifications@github.com> escreveu no dia quarta, 12/12/2018
à(s) 16:34:

**Comment by dsheiko** [comment]
Thanks Vanessa again. Now I get it. It was my bad, I didn't test typing in select after extending select option content. It's fixed now on master branch (https://github.com/dsheiko/puppetry/commit/3c784e3a4f91491573b6937480b4b9046589b756)
. The builds will be deployed for downloading with next release in a few days. Please just select with mouse, avoid typing in Page/Element Method selects till then. **Alternatively** you can build already manually the fixed version as :
```
git clone https://github.com/dsheiko/puppetry.git
cd puppetry
npm install 
npm run build
npm run dist:all
cd dist
```
in the `dist` subdirectory you will find the builds per platform

**Comment by vteixeira19** [comment]
I'm glad to have helped. Thanks for the fix.

Melhores cumprimentos.
Vanessa Teixeira


Dmitry Sheiko <notifications@github.com> escreveu no dia quinta, 13/12/2018
à(s) 10:44:

**Comment by dsheiko** [comment]
Eventually the release with the fix published https://github.com/dsheiko/puppetry/releases/tag/v1.0.5
Though it contains a lot of UI improvements.
BTW, since you are an actual QA engineer, do you have any ideas on how to make the UI for suitable for people without developer background? I'm wonder if I shall change command representation in the table to make it more human-readable
e.g. instead of 
```
page.goto("https://google.com")
```
something like
```
visit page by https://google.com
```
What do you think?

**Comment by vteixeira19** [comment]
Hello!

I appreciate the corrections, I’ve now checked them and now everything
works fine when writing on the inputs :)
I also noticed the UI improvments and believe it looks much better now.

I agree that it would be a good idea changing the 'page.goto' to 'visit
page', because it’s easier for someone that isn’t as tech savvy, or doesn’t
have much programming knowledge, to know that’s where he needs to copy the
URL to.

I took the liberty to double check the platform and I believe that there
are other improvements that also should be made, for example:

1 - When we finish a Test Run we get the results like this -
https://cl.ly/89c6847a7d0a - The way the results are structured the
'Failed' message appears mixed with other tests that didn’t failed. It
gives the impression a problem occurred in Tests that actually had no
problems. This alert be distinguished more in relation to the Test that
actually failed and give more details regarding the tests that failed,
showing the actual command that failed, so it’s easier to fix the problem.
Grouping the Tests that Failed and Tests that passed in separate areas
would also help.

2 - In this area - https://cl.ly/f30010c3276f - The text should be more
specific, as simply having 'Button' gives the impression you are setting
the location of the Button in the page and it’s not easy to understand it’s
actually referring to which button on the mouse to press.

3 - I believe it’s important to have more than one Project simultaneously.

4 - Whenever we create a new project, we need to immediately create a
‘Suite’, however there is no indication notifying us to do so, which may
leave the user to think that the platform isn’t working. Upon the creation
of the Project a Popup or something similar should appear with the
instructions that another action is necessary. After creating a new project
it currently looks like this - https://cl.ly/0322396aefa8

5 - Where does the Project appear? It looks like the program crashed and
did not respond.

6 - I would also like to point out that Opening a Project is not very
intuitive. I’ve only been able to open Suites -  https://cl.ly/0c0dcc69196e

I hope I have helped you :)

Best Regards,
Vanessa Teixeira


Dmitry Sheiko <notifications@github.com> escreveu no dia quarta, 19/12/2018
à(s) 09:03:
